[2022-06-05 10:33:17,729] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.controle_fluxo scheduled__2022-06-05T13:32:02.850037+00:00 [queued]>
[2022-06-05 10:33:17,737] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.controle_fluxo scheduled__2022-06-05T13:32:02.850037+00:00 [queued]>
[2022-06-05 10:33:17,738] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 10:33:17,738] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-05 10:33:17,738] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-05 10:33:17,750] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): controle_fluxo> on 2022-06-05 13:32:02.850037+00:00
[2022-06-05 10:33:17,756] {standard_task_runner.py:52} INFO - Started process 71439 to run task
[2022-06-05 10:33:17,760] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'controle_fluxo', 'scheduled__2022-06-05T13:32:02.850037+00:00', '--job-id', '860', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp_4vtsfei', '--error-file', '/tmp/tmp22pvfqwr']
[2022-06-05 10:33:17,760] {standard_task_runner.py:80} INFO - Job 860: Subtask controle_fluxo
[2022-06-05 10:33:17,819] {task_command.py:370} INFO - Running <TaskInstance: census.controle_fluxo scheduled__2022-06-05T13:32:02.850037+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-05 10:33:17,896] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=controle_fluxo
AIRFLOW_CTX_EXECUTION_DATE=2022-06-05T13:32:02.850037+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-05T13:32:02.850037+00:00
[2022-06-05 10:33:18,174] {python.py:173} INFO - Done. Returned value was: None
[2022-06-05 10:33:18,190] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=controle_fluxo, execution_date=20220605T133202, start_date=20220605T133317, end_date=20220605T133318
[2022-06-05 10:33:18,213] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-05 10:33:18,257] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
